### PR TITLE
Tarrgona(ES): Add Camp Tarragona as child of Tarragona metastation

### PR DESCRIPTION
**Pull request by paulRbr**
"Camp Tarragona" is the station where high speed trains arrive in Tarragona. It's part of the metastation "Tarragona" city.

### CODE REVIEW

File: stations.csv [hunk 1]
```diff
@@ -22435,7 +22435,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 24441;Tarancon;tarancon;7166006;;40.006402;-3.009831;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;7100233;f;ezjged;t;;f;;f;;f;;f;;f;;;;f;;f;66006;t;;f;;f;;f;f;;;;;;;;;;;;;;;;Таранкон;;;塔兰孔
 24442;Tardelcuende;tardelcuende;7184103;;41.593501;-2.639161;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;7184103;f;;f;;f;;f;;f;;f;;f;;;;f;;f;84103;t;;f;;f;;f;f;;;;;;;;;;;;;;;;Тарделькуэнде;;;塔尔德尔昆德
 24443;Tardienta;tardienta;7178200;;41.975777;-0.538067;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;7100117;f;;f;;f;;f;;f;;f;;f;;;;f;;f;78200;t;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
-24444;Tarragona;tarragona;;;41.11886;1.243387;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;sp2cum;t;;f;;f;;f;;f;;f;;;;f;;f;TARRA;t;;f;;f;;f;f;;Alle Bahnhöfe;All stations;Todas las estaciones;Tarragone toutes gares;Tutte le stazioni;;;;;;タラゴナ;;;;Таррагона;;;
+24444;Tarragona;tarragona;;;41.11886;1.243387;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;sp2cum;t;;f;;f;;f;;f;;f;;;;f;;f;TARRA;t;;f;;f;;f;f;;;;;Tarragone toutes gares;;;;;;;;;;;;;;
```
Comment (mckbrmn) [inline stations.csv]: Why did you remove all but the French?

Comment (paulRbr) [reply stations.csv]: @mckbrmn because I know only the French translation of `Tarragona`

Comment (paulRbr) [reply stations.csv]: and the other `Tarragona estacio` has an fr translation to `Tarragone` so I need to differentiate the metastation name in French
